GliaWidgets/Sources/View/Chat/Message/GvaPersistentButtonOptionView.swift

```diff
@@ -0,0 +1,67 @@
+import UIKit
+
+class GvaPersistentButtonOptionView: BaseView {
+    var tap: (() -> Void)?
+
+    private let text: String?
+    private let textLabel = UILabel()
+    private let choiceButton = UIButton()
+    private let viewInsets = UIEdgeInsets(top: 10, left: 12, bottom: 10, right: 12)
+
+    init(text: String?) {
+        self.text = text
+        super.init()
+    }
+
+    @available(*, unavailable)
+    required init() {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func setup() {
+        backgroundColor = .clear
+        layer.backgroundColor = UIColor.white.cgColor
+        layer.cornerRadius = 4
+        heightAnchor.constraint(equalToConstant: 42).isActive = true
```

Comment:
all stuff related to establishing constraints should happen in `defineLayout` (it happens after updating constraints), so all of them will be constructed correctly. Please move this line to defineLayout

Reply:
fixed [4720b0e](https://github.com/salemove/ios-sdk-widgets/pull/683/commits/4720b0ee106d7a116f5fdceef08605e241372e8f)